### PR TITLE
Fix for exporting map image to the clipboard and modifications per previous PR code review

### PR DESCRIPTION
**Pull request by FJBDev**

### CODE REVIEW

File: bundles/net.tourbook/src/net/tourbook/map2/action/ActionExportMapViewClipboard.java [hunk 1]
```diff
@@ -43,22 +50,40 @@ public ActionExportMapViewClipboard(final Map2View mapView) {
...
+         //Workaround for the issue on Windows where the map is missing but the legend
+         //is displayed (GIMP and Inkscape). No issue when sing MS Paint
+
+         try {
+            final String absoluteFilePath = Files.createTempFile("map", ".jpg").toString(); //$NON-NLS-1$ //$NON-NLS-2$
+
+            //We export the image to a file as a JPEG image
+            final ImageLoader loader = new ImageLoader();
+            loader.data = new ImageData[] { mapViewImage.getImageData() };
+            loader.save(absoluteFilePath, SWT.IMAGE_JPEG);
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/map2/action/ActionExportMapViewClipboard.java:71]: PNG should be used, a lossless format

**Comment by wolfgang-ch** [comment]
After merging into main, I did some smaller modifications, see my comments in the commits

51a2c9713d194db42b4f4635fa29898c68ff79a9
dc5931d8a917e1532ef9949afc7db20bfb9309d7
f3f9cfe35fc4eb8790cf6c6fbe1cd4b940b5977b
859a76b1e57164f1999dbb2c4f106412331027b1

File: bundles/net.tourbook/src/net/tourbook/map2/action/ActionExportMapViewClipboard.java (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/map2/action/ActionExportMapViewClipboard.java:71]: I see you have updated that in main

**Comment by FJBDev** [comment]
> 
> 
> After merging into main, I did some smaller modifications, see my comments in the commits
> 
> [51a2c97](https://github.com/wolfgang-ch/mytourbook/commit/51a2c9713d194db42b4f4635fa29898c68ff79a9)
> [dc5931d](https://github.com/wolfgang-ch/mytourbook/commit/dc5931d8a917e1532ef9949afc7db20bfb9309d7)
> [f3f9cfe](https://github.com/wolfgang-ch/mytourbook/commit/f3f9cfe35fc4eb8790cf6c6fbe1cd4b940b5977b)
> [859a76b](https://github.com/wolfgang-ch/mytourbook/commit/859a76b1e57164f1999dbb2c4f106412331027b1)

Looks good to me

**Pull request merged**